homeassistant/components/tuya/binary_sensor.py

```diff
@@ -256,6 +256,23 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
         ),
         TAMPER_BINARY_SENSOR,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        TuyaBinarySensorEntityDescription(
+            key=f"{DPCode.WORK_STATE}_manual",
+            dpcode=DPCode.WORK_STATE,
+            name="Manual",
+            icon="mdi:information",
+            on_value="manual",
+        ),
+        TuyaBinarySensorEntityDescription(
+            key=f"{DPCode.WORK_STATE}_auto",
+            dpcode=DPCode.WORK_STATE,
+            name="Auto",
+            icon="mdi:information",
+            on_value="auto",
+        ),
```

Comment:
Should this be a sensor with an enum device class instead?

Reply:
My coding foo is not strong, could you please an example?

Reply:
It is fairly new, so no example in the Tuya integration yet. Create an entity description and set the "enum" device class, additionally, we need to set the "options" parameter in the description to a list of possible options this sensor can provide.